doc/release-notes.md

```diff
@@ -117,6 +117,11 @@ deprecated in V0.15.1, and has now been removed. Miners should use the
 `-blockmaxweight` option if they want to limit the weight of their blocks'
 weights.
 
+Python Support
+--------------
+
+Support for Python 2 has been discontinued for all test files and tools.
```

Comment:
NACK based on my earlier feedback in this pull request.

Reply:
I disagree. We should remove support for a legacy language which will soon not be maintained, and which may hide subtle bugs due to incompatibilities with the new language.

What do we gain by claiming to support python2?

Reply:
I am not saying we should claim to support python2, but we should not claim that python2 support has been discontinued for "all test files" (specifically *util tests*, since the functional tests are already on python3). I think it is reasonable to support python2 util tests until we need to switch off of it because it would otherwise be a burden to continue to support python2. I don't see why we should put effort into making the util tests incompatible with python2 when we might as well leave them supported on a best effort basis.

Reply:
@MarcoFalke - Honestly, supporting on a "best effort basis" seems like a nightmare waiting to happen. Person A introduces a commit that breaks Py2 compatibility, Person B complains that unit tests are failing, Persons C-? have to deal with the whole thing. I think having 2 & 3 supported is good for transition purposes, and making sure that nothing is broken during a switch. Beyond that, I'm with @jnewbery and don't see any real gain from claiming to support Py2. At most, I'd be okay with a statement mentioning that, while Py2 compatibility is best effort, people *must* use Py3 before complaining that unit tests fail.

Reply:
I think it is not worth arguing about that.